Source/Core/Common/StringUtil.cpp

```diff
@@ -25,6 +25,8 @@
 
 #ifdef _WIN32
 #include <Windows.h>
+constexpr u32 CODEPAGE_SHIFT_JIS = 932;
+constexpr u32 CODEPAGE_LATIN1 = 1252;
```

Comment:
I don't think calling this codepage `LATIN1` is a good idea, as it might make people think that it refers to ISO-8859-1. The proper name for this codepage is Windows-1252.